packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:math' show max;
+
+import '../globals.dart';
+import 'io.dart' as io;
+import 'terminal.dart';
+
+const String fire = '🔥';
+const int maxLineWidth = 84;
+
+/// Encapsulates the help text construction and printing
+class CommandHelp {
+
+  CommandHelp._(this.key, this.description, [this.inParenthesis = '']);
+
+  factory CommandHelp.L() => CommandHelp._('L', 'Dump layer tree to the console.', 'debugDumpLayerTree');
```

Comment:
Rather than use these different factor constructors, I think it would be more idiomatic to create static const values, such as

```
static const CommandHelp L = CommandHelp._('L', '...');
```

Then you could use it with one fewer set of parens `CommandHelp.L.print();`,

Reply:
That's a good idea! The reason I had them as factory methods was to lazy initialize the different commands, but I guess it doesn't really matter as this is only displayed once.